SETI/rms-opus
# Support COUVIS calibration files in download data

- Were any Django, import pipeline, table_schema, or dictionary files modified? Y
  - Database used: opus3_test_200211
    - Need to run the following two imports:
      - python main_opus_import.py --import-dictionary
      - python main_opus_import.py --do-all-import --import-fake-images COUVIS_0002
  - All Django tests pass: Y
- Were any JavaScript or CSS files modified? N
  - JSHINT run on all affected files: NA
  - Tested on Chrome, Firefox, Safari, and iOS: Y
    (Remember to test all browser sizes)
  - Tested for race conditions (with delayed API calls if applicable): NA

Description of changes:
  - Add tooltip definition for COUVIS calibration files in download data.
  - Update the test case to have the correct expected return of COUVIS opus products.

Known problems:
None
